Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM neighborhoods/php-fpm-phalcon:php7.2_phalcon3.4
```

Comment:
potential blocker: `phpunit-first-steps` is a public repo, part of the goal being that we share the tutorial's content with the broader world. it may make this content more generally accessible to set this repo up with a docker environment that is more of a vanilla open source configuration.

relying on mason is basically a non-starter in that context.

something super duper simple (and tiny) like this might do the trick, since this repo only requires php right now:

```dockerfile
FROM php:7.2.13-cli-alpine

ADD . ~/app

WORKDIR ~/app

RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
RUN php -r "if (hash_file('sha384', 'composer-setup.php') === '93b54496392c062774670ac18b134c3b3a95e5a5e5c8f1a9f115f203b75bf9a129d5daa8ba6a13e2cc8a1da0806388a8') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;"
RUN php composer-setup.php
RUN php -r "unlink('composer-setup.php');"

RUN mv composer.phar /usr/local/bin/composer

RUN composer install
```

...and then for the docker-compose.yml:

```yml
version: '3'
services:
  app:
    build: .
    volumes:
      - .:/app
```

Reply:
Good point, I forgot about this being available to those who don't have access to Mason. I copied your 2 examples instead, I only had to add `tty: true` to docker-compose.yml to be able to shell into the container.